### PR TITLE
docs: document wordBreaker joinWordsAt property

### PR DESCRIPTION
**Pull request by eddieantonio**
Documents how to customize word breaking with `joinWordsAt:` property.

Depends on:
 - keymanapp/keyman#3021
 - keymanapp/keyman#3032

~~Closes~~ Addresses: keymanapp/keyman#2753

### CODE REVIEW

File: developer/14.0/guides/lexical-models/advanced/word-breaker.php [hunk 1]
```diff
@@ -32,17 +32,73 @@
...
+    joinWordsAt: ['-', ':', '@'], // join words at hyphens, colons, at-signs
+  }
+};
+
+export default source;</code></pre>
+
+
+<h2 id="custom"> Writing a custom word breaking </h2>
+
+<p> the word breaker function can be specified in the
```
Comment (darcywong00) [inline developer/14.0/guides/lexical-models/advanced/word-breaker.php:89]:
```suggestion
<p> The word breaker function can be specified in the
```

File: developer/14.0/guides/lexical-models/advanced/word-breaker.php [hunk 2]
```diff
@@ -32,17 +32,73 @@
...
 text to determine where the words are. </p>
 
-<p> The word breaker function can be specified in the
+<p> You can customize the word breaker in two ways: </p>
+
+<ul>
+  <li> If the default word breaker creates <strong>too many splits</strong>,
+       you can <a href="#join">choose which strings join words together</a>. </li>
+  <li> If the default word breaker creates <strong>not enough splits</strong>,
+       you must <a href="#custom">create your own word breaking function</a>. </li>
```
Comment (darcywong00) [inline developer/14.0/guides/lexical-models/advanced/word-breaker.php:41]: To be consistent with the rest of the page, maybe use "word breaker function" here and line 87.

File: developer/14.0/guides/lexical-models/advanced/word-breaker.php [hunk 3]
```diff
@@ -32,17 +32,73 @@
...
+    use: 'default',
+    // CUSTOMIZE THIS:
+    joinWordsAt: ['-', ':', '@'], // join words at hyphens, colons, at-signs
+  }
+};
+
+export default source;</code></pre>
+
+
+<h2 id="custom"> Writing a custom word breaking </h2>
```
Comment (darcywong00) [inline developer/14.0/guides/lexical-models/advanced/word-breaker.php:87]:
```suggestion
<h2 id="custom">Writing a custom word breaker function</h2>
```

**Comment by eddieantonio** [comment]
> Darcy's suggestions should be addressed and then LGTM

Okay: I'll wait until we've merged keymanapp/keyman#3021 and keymanapp/keyman#3032 before merging this, because this PR is the end-user documentation of those two Keyman PRs.

**Comment by mcdurdin** [comment]
> Okay: I'll wait until we've merged [keymanapp/keyman#3021](https://github.com/keymanapp/keyman/pull/3021) and [keymanapp/keyman#3032](https://github.com/keymanapp/keyman/pull/3032) before merging this, because this PR is the end-user documentation of those two Keyman PRs.

It's okay to go ahead and merge this because it's the 14.0 documentation which is always understood to be in flux. It won't appear by default until we release 14.0 (users have to manually navigate to the newer version).

**Pull request merged**